src/qibocal/protocols/characterization/rabi/utils.py

```diff
@@ -116,3 +116,82 @@ def plot(data, qubit, fit):
     figures.append(fig)
 
     return figures, fitting_report
+
+
+def plot_proba(data, qubit, fit):
```

Comment:
```suggestion
def plot_probabilities(data, qubit, fit):
```
This name should be better for readability